doc/howto/formatting.rst

```diff
@@ -0,0 +1,87 @@
+How to Set up Automatic Formatting
+==================================
+
+This guide will show you how to configure Dune so that it can check formatting
+of your source code.
+
+Formatting is defined per project. This ensures that if a project is reused
+elsewhere, its formatting configuration will not interfere.
+
+Setting Up the Environment
+--------------------------
+
+First, let's open the ``dune-project`` file. Make sure that the version
+specified in ``(lang dune X.Y)`` is at least ``2.0``. Most formatting
+configuration happens in that file. If you want to format OCaml sources and
+``dune`` files, you don't have anything to add. Otherwise, refer to the
+:ref:`(formatting) <formatting>` stanza.
+
+Then, we need to install some code formatting tools. For OCaml code, this means
+installing OCamlFormat_ with ``opam install ocamlformat``. Formatting ``dune``
+files is built into Dune and does not require extra tools. For Reason code,
+this uses the ``refmt`` tool which is already installed if you are using Reason
+syntax in your project. If your project uses a :term:`dialect`, a specific tool
+might be required.
+
+.. _ocamlformat: https://github.com/ocaml-ppx/ocamlformat
+
+OCamlFormat requires some configuration. Take note of the version returned by
```

Comment:
In the ocamlformat readme they spell it all as lowercase. I have not seen this capitalised version before.

Reply:
Since it might be awkward to put this at the start of the sentence you could say:
```suggestion
Using ocamlformat requires some configuration. Take note of the version returned by
```

Reply:
That used to be the official spelling. I'll ask around.

Reply:
I got confirmation from the maintainer that it's still OCamlFormat for the tool and `ocamlformat` for the command.